### PR TITLE
link phase save output: manifest might contain multiple items with same path

### PR DESCRIPTION
**Pull request by ansyral**
@herohua

### CODE REVIEW

**Comment by hellosnow** [comment]
:shipit:

File: src/Microsoft.DocAsCode.Build.Engine/LinkPhaseHandlerWithIncremental.cs [hunk 1]
```diff
@@ -235,7 +240,7 @@ select new
                             }
 
                             File.Copy(fullPath, Path.Combine(IncrementalContext.BaseDir, fileName));
-                            CurrentBuildVersionInfo.BuildOutputs.Add(path, fileName);
```
Comment (herohua) [inline src/Microsoft.DocAsCode.Build.Engine/LinkPhaseHandlerWithIncremental.cs]: don't think this line needs update

File: src/Microsoft.DocAsCode.Build.Engine/LinkPhaseHandlerWithIncremental.cs [hunk 2]
```diff
@@ -214,6 +214,11 @@ select new
                     }
                     foreach (var path in items)
                     {
+                        // for redirection, path might be duplicate
```
Comment (herohua) [inline src/Microsoft.DocAsCode.Build.Engine/LinkPhaseHandlerWithIncremental.cs]: update the comment here

File: src/Microsoft.DocAsCode.Build.Engine/LinkPhaseHandlerWithIncremental.cs (hunk 1 above)
Comment (ansyral) [inline src/Microsoft.DocAsCode.Build.Engine/LinkPhaseHandlerWithIncremental.cs]: double assurance

**Pull request merged**